### PR TITLE
#7711: Created custom module with a block that provides weather in ce…

### PR DESCRIPTION
**Pull request by aster06**
…rtain city

### CODE REVIEW

**Review by ivandudardb — APPROVED** [review]
LGTM

File: web/modules/custom/weather/src/Plugin/Block/WeatherBlock.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+)]
+class WeatherBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $city = 'Lutsk';
+    $key = '00be8ffdef324d4ca2c112336241403';
+    $res = file_get_contents('http://api.weatherapi.com/v1/current.json?key=' . $key . '&q=' . $city);
```
Comment (lavrenchuk-dev) [inline web/modules/custom/weather/src/Plugin/Block/WeatherBlock.php:25]: Next time use \Drupal::httpClient please

File: web/modules/custom/weather/templates/weather.html.twig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="weather-class">
+  {{ temp }}
+</div>
```
Comment (lavrenchuk-dev) [inline web/modules/custom/weather/templates/weather.html.twig:3]: Such a simple template could be achieved by #container render element
But it is fine if you planning to extend this in future.

File: web/modules/custom/weather/weather.info.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+name: Weather
+type: module
+description: 'The best weather module for Drupal'
```
Comment (lavrenchuk-dev) [inline web/modules/custom/weather/weather.info.yml:3]: there is no evidence

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**